RosettaCore/src/main/java/com/hubspot/rosetta/Rosetta.java

```diff
@@ -32,11 +32,13 @@ public static void setMapper(ObjectMapper mapper) {
     INSTANCE.set(mapper);
   }
 
-  public static ObjectMapper cloneAndCustomize(ObjectMapper mapper) {
+  public static ObjectMapper cloneAndCustomize(ObjectMapper mapper, List<Module> modules) {
```

Comment:
Thanks for pointing this out, @mjball. Thinking about this helped me find a solution that is both backwards compatible and, I think, much cleaner.